aws/sqs/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "stack" {
+  description = "The name of the stack"
+  type        = string
+}
+
+variable "env" {
+  description = "The name of the environment"
+  type        = string
+}
+
+variable "queue_name" {
```

Comment:
nit, i've been calling these `identifier` to be less ambiguous, though your description does do a decent job clarifying, so.. 🤷‍♂️